src/actionModifiers/ImportExportWithDatasets/importWithDatasetsModifier.js

```diff
@@ -169,10 +169,12 @@ function importWithDatasetsModifier(openmct) {
 
         Object.values(openmct)
             .forEach(object => object.composition
-                ?.forEach(identifier => {
+                ?.forEach( identifier => {
                     if (referencesDataset(identifier)) {
-                        const datasetIdentifier = getDatasetIdentifier(identifier);
```

Comment:
I think you want to delete the original `getDatasetIdentifier` if it's not being used anywhere else.